src/styles/Booking.styled.js

```diff
@@ -0,0 +1,135 @@
+import styled from "styled-components";
+import { ButtonStyle } from "../components/Button/Button.styled";
+export const BookingStyles = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+
+  ${ButtonStyle} {
```

Comment:
:+1: This is how we should be using styled components to style nested components.